docs/content/connector/connector_protocol.md

```diff
@@ -78,12 +76,11 @@ Parameters can be specified during connector publication.
 The connector has to respond with one of the following status codes: **201**, **202**, **401** or **403**.
 Optionally the response can contain a payload specifying further steps required for the installation.
 The content of field **furtherStep** will be shown in the [Developer Center](https://devcenter.connctd.io/), formatted according to its type.
```

Comment:
```suggestion
The content of field **furtherStep** will be shown in the [Developer Center](https://devcenter.connctd.io/), formatted according to its type. It can become handy if the installation requires additional steps performed by the user (e.g. giving consent during OAuth2 flow)
```